src/components/PostPageDetail/index.js

```diff
@@ -561,7 +561,7 @@ const PostPageDetailIdComponent = (props) => {
         <React.Fragment>
           <Header
             isPostDetail={true}
-            hideThreeDot={true}
+            hideThreeDot={false}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `hideThreeDot` prop in the `Header` component has been changed from `true` to `false`. This change will make the three-dot menu visible in the header. Ensure this is the intended behavior.

```diff
-            hideThreeDot={true}
+            hideThreeDot={false}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->